Cargo.toml

```diff
@@ -26,9 +32,10 @@ alloy = { git = "https://github.com/alloy-rs/alloy", features = [
 
 # zk-evm dependencies
 plonky2 = "0.2.2"
-evm_arithmetization = { git = "https://github.com/0xPolygonZero/zk_evm.git", tag = "v0.3.1" }
-trace_decoder = { git = "https://github.com/0xPolygonZero/zk_evm.git", tag = "v0.3.1" }
-proof_gen = { git = "https://github.com/0xPolygonZero/zk_evm.git", tag = "v0.3.1" }
+evm_arithmetization = { git = "https://github.com/fractal-zkp/zk_evm.git", branch = "feat/partial_trie_builder" }
+mpt_trie = { git = "https://github.com/fractal-zkp/zk_evm.git", branch = "feat/partial_trie_builder" }
+trace_decoder = { git = "https://github.com/fractal-zkp/zk_evm.git", branch = "feat/partial_trie_builder" }
+proof_gen = { git = "https://github.com/fractal-zkp/zk_evm.git", branch = "feat/partial_trie_builder" }
```

Comment:
You can use latest release here `v0.4.0`. As we will merge `zero-bin` and `zk_evm`, we will probably switch to current local repo path reference soon.